ONSdigital/census-rm-toolbox
# Use shutil to move processed queue dump files

# Checklist
Reminder: If any change made to this repo affects the acceptance tests then the version pin in the Pipfile in [census-rm-acceptance-tests]() needs to be updated.
* [ *] Updated census-rm-acceptance-tests version pin (if applicable)

# Motivation and Context
The dumping files to queue script was erroring in GCP when attempting to move a file.  Using shutil.move instead solves it.

# What has changed
Using shutil.move to move the processed queue dump file.

# How to test?
- Set up a Rabbit queue with a few dummy messages (really doesn't matter what).
- Spin up the toolbox
- Create the initial queue dump file by running `dumpqueuetofiles <queue_name> <number of messages> <output directory>`
- Check the queue is empty
- Dump the file back to the queue by running ` dumpfilestoqueue <queue_name> <input directory> <output directory>`
- Check that the messages are back on the queue

# Links
- https://trello.com/c/rNsSS5Mj
